src/app/frontend/typings/root.shared.ts

```diff
@@ -25,3 +25,7 @@ export enum SupportedResources {
   Secret = 'Secret',
   PersistentVolumeClaim = 'PersistentVolumeClaim',
 }
+
+export interface StringMap {
+  [key: string]: string;
+}
```

Comment:
What is reason for moving this from `root.api.ts` to `root.shared.ts`?

Reply:
Because `StringMap` is now needed on `src/app/frontend/typings/volume.api.ts`, hence it's a shared resource now. More specifically, for the class `CSIVolumeSource` on the variable `volumeAttributes`.